src/pages/home/index.jsx

```diff
@@ -0,0 +1,255 @@
+import Project from "./../projects/index";
+
+export default class Home extends Component {
+  state = {
+    reposData: [],
+    name: "",
+    username: "",
+    image: "",
+    bio: "",
+    company: "",
+    location: "",
+    email: "",
+    website: "",
+    twitterUserName: "",
+    followers: 0,
+    following: 0,
+  };
+
+  componentDidMount = () => {
+    this.fetchUserData();
+    this.fetchReposData();
+    this.fetchStarred();
+  };
+
+  fetchUserData = async () => {
+    const response = await fetch("https://api.github.com/users/saritadc");
+    const data = await response.json();
+
+    this.setState({
+      name: data.name,
+      userName: data.login,
+      image: data.avatar_url,
+      bio: data.bio,
+      company: data.company,
+      location: data.location,
+      email: data.email,
+      website: data.blog,
+      twitterUserName: data.twitter_username,
+      followers: data.followers,
+      following: data.following,
+    });
+  };
+
+  fetchReposData = async () => {
+    const response = await fetch("https://api.github.com/users/saritadc/repos");
+    const data = await response.json();
+
+    this.setState({
+      reposData: data,
+      originalReposData: data,
+    });
+  };
+
+  fetchStarred = async () => {
+    const response = await fetch(
+      "https://api.github.com/users/saritadc/starred"
+    );
+    const data = await response.json();
+    this.setState({ star: data.length });
+  };
+
+  handleOnEdit = () => {
+    this.setState({ isEditMode: true });
+  };
+
+  handleOnCancel = () => {
+    this.setState({ isEditMode: false });
+  };
+
+  handleOnSubmit = (event) => {
+    const { bio, company, location, email, website, twitterUserName } =
+      this.state;
+    event.preventDefault();
+    fetch("https://api.github.com/user", {
+      method: "PATCH",
+      headers: new Headers({
+        Authorization: "Bearer " + "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",
+        "Content-Type": "application/json",
+      }),
+      body: JSON.stringify({
+        bio,
+        company,
+        location,
+        email,
+        website,
+        twitterUserName,
+      }),
+    })
+      .then((response) => response.json())
+      .then((data) => console.log(data));
+
+    this.setState({
+      isEditMode: false,
+    });
+  };
+
+  handleOnChange = (event) => {
+    this.setState({
+      [event.target.id]: event.target.value,
+    });
+  };
+
+  render() {
+    const {
+      reposData,
+      name,
+      userName,
+      image,
+      bio,
+      company,
+      location,
+      email,
+      website,
+      twitterUserName,
+      followers,
+      following,
+      isEditMode,
+      star,
+    } = this.state;
+
+    const { handleOnEdit, handleOnCancel, handleOnSubmit, handleOnChange } =
+      this;
+
+    return (
+      <div>
+        <nav className="nav-bar">
+          <ul className="nav-bar__list">
+            <Link to={"/repos"} className="nav-bar__list-link">
```

Comment:
Please use route constants 
![image](https://user-images.githubusercontent.com/22271897/120759116-634d9e80-c532-11eb-9116-99eaa059ce35.png)

Import it like this 
![image](https://user-images.githubusercontent.com/22271897/120759304-9a23b480-c532-11eb-84c7-137a5a4570b4.png)

PS: create a folder to keep constants like this: 
![image](https://user-images.githubusercontent.com/22271897/120759398-b4f62900-c532-11eb-92a5-2b712b506015.png)